### PR TITLE
Hotfix/lara lens prefix value from env

### PR DESCRIPTION
**Pull request by JexPY**
Hello, that's my first pr on public repo I know that's not hard one but ready for any critic thanks.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -153,7 +153,7 @@ With the following configuration you will enable the web view (_web-enabled_ par
 
 ```php
 return [
-    'prefix' => 'laralens', // URL prefix (default=laralens)
+    'prefix' => env('LARALENS_WEB_ENABLED', 'laralens'), // URL prefix (default=laralens)
```
Comment (roberto-butti) [inline README.md]: for 'prefix', the env variable needs to be. LARALENS_PREFIX and not LARALENS_WEB_ENABLED

File: config/config.php [hunk 2]
```diff
@@ -4,7 +4,7 @@
  * LaraLens Configuration.
  */
 return [
-    'prefix' => 'laralens', // URL prefix (default=laralens)
+    'prefix' => env('LARALENS_WEB_ENABLED', 'laralens'), // URL prefix (default=laralens)
```
Comment (roberto-butti) [inline config/config.php]: for 'prefix', the env variable needs to be. LARALENS_PREFIX and not LARALENS_WEB_ENABLED

**Review by roberto-butti — CHANGES_REQUESTED** [review]

File: config/config.php (hunk 2 above)
Comment (JexPY) [reply config/config.php]: Thanks. Sorry.

**Review by JexPY — COMMENTED** [review]
Sorry for my mistake I changed the string.

**Review by roberto-butti — APPROVED** [review]

**Pull request merged**